src/main/ts/components/useData.ts

```diff
@@ -65,41 +67,47 @@ export default function useData(): UseDataReturn {
 
   useEffect(() => {
     (async () => {
-      const issueRepartitionData = await getIssues();
+      try {
+        const issueRepartitionData = await getIssues();
 
-      let cumulativeIssueCount = 0;
-      const cumulativeData = issueRepartitionData.map(({ x, y }) => {
-        cumulativeIssueCount += y;
-        return {
-          x,
-          y: cumulativeIssueCount,
-        };
-      });
+        setHasRequestFailed(false);
```

Comment:
This is not necessary, is it?

Reply:
Oh indeed. As this is happening only on page load, it will be ran only once, so we can't have a case where there was a previous error state we have to reset, good catch